### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by maxicms** [dingjie/atom-sync#1]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.208.0
**System**: Mac OS X 10.10.4
**Thrown From**: [atom-sync](https://github.com/dingjie/atom-sync) package, v0.4.1

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:759

TypeError: undefined is not a function
    at Object.module.exports.ConfigHelper.getCurrentProjectDirectory (/Users/admin/.atom/packages/atom-sync/lib/helper/config-helper.coffee:71:62)
    at Object.module.exports.ConfigHelper.getFullPath (/Users/admin/.atom/packages/atom-sync/lib/helper/config-helper.coffee:59:19)
    at Object.module.exports.ConfigHelper.load (/Users/admin/.atom/packages/atom-sync/lib/helper/config-helper.coffee:20:19)
    at Object.module.exports.ServiceController.uploadEditingFile (/Users/admin/.atom/packages/atom-sync/lib/controller/service-controller.coffee:28:28)
    at /Users/admin/.atom/packages/atom-sync/lib/atom-sync.coffee:62:37
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
    at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1016:20)
    at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:998:19)
    at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:579:26)
    at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:523:18)
    at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:506:19)
    at Workspace.module.exports.Workspace.saveActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:585:35)
    at atom-workspace.atom.commands.add.core:save (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace-element.js:317:30)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
    at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)
```

### Commands

```
     -4:52.4.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -4:52.4.0 editor:newline (atom-text-editor.editor.is-focused)
     -4:52.3.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -4:52.3.0 editor:newline (atom-text-editor.editor.is-focused)
     -4:52.1.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -4:52.1.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:02.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:01.5.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "one-dark-dim-syntax"
    ],
    "disabledPackages": [
      "atom-css-class-checker",
      "atom-css-class-checker",
      "atom-css-class-checker",
      "atom-css-class-checker"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Remote-FTP, v0.7.1
atom-sync, v0.4.1
autocomplete-php, v0.3.6
browser-plus, v0.0.23
css-snippets, v0.8.0
emmet, v2.3.10
file-icons, v1.5.7
language-smarty-php, v0.5.0
linter-php, v0.0.15
linter-phpcs, v0.0.15
one-dark-dim-syntax, v0.1.0
php-introspector, v1.0.0
save-session, v0.15.5
tool-bar, v0.1.7
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by dingjie** [comment]
Hi maxicms,
I wonder if the tree view panel was closed when exception was thrown

**Comment by maxicms** [comment]
exception was thrown in save file

**Comment by dingjie** [comment]
When you saving the file, is the left side panel (tree view) visible ?
![image](https://cloud.githubusercontent.com/assets/586262/8087101/8b31ffa4-0fcb-11e5-8a5b-266023554cfb.png)

**Comment by maxicms** [comment]
tree view  visible  :)

**Comment by dingjie** [comment]
Thanks maxicms, I'll be working on this soon.

**Comment by dingjie** [comment]
I've just released a new version 0.4.2, would you please try if this version can resolve your problem ?

**Comment by dingjie** [comment]
should be resolved in [0.4.3](https://github.com/dingjie/atom-sync/releases/tag/v0.4.3)

**Comment by mishoboss** [comment]
I just reproduced it on the latest Atom version - 1.0.5. Version of atom-sync is 0.5.2.
